src/powfacpy/base_interface.py

```diff
@@ -657,6 +658,22 @@ def export_to_csv(self,
       else:
         self.format_csv_for_comtrade(path)
 
+  def export_to_pandas(self, result_objects, elements, variables):
```

Comment:
Add docstring.
Should file path and name be (optional) arguments?

Reply:
We should consider moving the method to results_interface.py because it is rather specific and the base interface is already pretty large. I know that export_to_csv is also in base interface but that was added before the results interface was added.

Reply:
The method export_to_csv formats the headers when exporting from an ElmRes by default. This significantly affects the performance because it has to rewrite the whole file. When exporting to pandas, the csv file is just a step to get to pandas. We should consider formatting the headers in pandas instead of in the csv file which is probably much more performant than.

Reply:
filepath and filename can be hard coded here imo, because the file gets deleted directly after the pandas dataframe is created.